ui-admin/src/test-utils/mocking-utils.tsx

```diff
@@ -563,16 +567,31 @@ export const MOCK_SPANISH_LANGUAGE = {
   id: '1'
 }
 
+export type RenderInPortalRouterOpts = {
+  envName?: string,
+  mockWindowAlert?: boolean,
+  adminUsers?: AdminUser[], // will be put in the AdminUserContext
+  user: AdminUser, // will be put in the UserContext
+  permissions?: string[] // convenience for setting permissions without setting a full user with portalPermissions
+}
+
+export const defaultRenderOpts = {
+  envName: 'sandbox',
+  mockWindowAlert: true,
+  adminUsers: [],
+  user: mockAdminUser(false),
```

Comment:
This is a great change. Totally agree with testing as a non-superuser by default going forward!